contrib/win32/win32compat/shell-host.c

```diff
@@ -39,6 +39,19 @@
 #include "misc_internal.h"
 #include "inc\utf.h"
 
+/* Terminal Types */
+#define TERM_TYPE_ANSI    "ansi"
+#define TERM_TYPE_VT100   "vt100"
+#define TERM_TYPE_XTERM   "xterm"
+
+#define TT_ANSI     0x0001        /* Codes in all ANSI like terminals. */
+#define TT_VT100    0x0002        /* VT100 */
+#define TT_XTERM    0x0004        /* Xterm */
+
+#define TTM_VT100X   (TT_VT100 | TT_XTERM)
```

Comment:
I know it occurs when TERM is set to xterm...
We plan to take the conhost (xterm support) provided from windows 10 onwards.. so the end user can set the TERM to xterm.. It is especially required when he is working on cloud environment where he can ssh to any unix/windows machines from the same terminal by setting the TERM to xterm... This has to be supported and fixed with no regressions..